src/assets/history.ts

```diff
@@ -708,4 +714,9 @@ export const meetings: Meeting[] = [
       "title": "Video Game Club 54.0",
       "games": ["jusant", "cocoon"]
     },
+    {
+      "date": "2024-02-28",
```

Comment:
date typo -- 01-28

Reply:
my brain refused to believe we could have two meetings in one month!